Makefile

```diff
@@ -136,12 +136,16 @@ install:
 	cp ./assets/wall_4K.png ${PREFIX}/share/hyprland
 	cp ./assets/wall_8K.png ${PREFIX}/share/hyprland
 
+	install -Dm644 -t ${PREFIX}/share/man/man1 ./docs/*.1
```

Comment:
umm this is not a PKGBUILD

also mkdir -p would be good

Reply:
`install -D` is mkdir + cp. I can drop the `-m644` if you want

Reply:
oh, didnt know, alr. I hope that u tested it

Reply:
Any FreeBSD user, if exists, is free to submit a PR to change it